JSDemos/Demos/Popup/Scrolling/jQuery/index.js

```diff
@@ -1,22 +1,80 @@
 $(() => {
-  $('#popup').dxPopup({
-    width: 550,
-    height: 350,
-    visible: true,
-    showTitle: false,
-    hideOnOutsideClick: false,
-    contentTemplate() {
-      const scrollView = $('<div />');
-
-      scrollView.append($('<img />', { src: '../../../../images/Popup-Scrolling-Image.jpg', class: 'center' }));
-      scrollView.append($('<div id="textBlock" />').html(longText));
-
-      scrollView.dxScrollView({
-        width: '100%',
-        height: '100%',
-      });
+  $('#show-scrollable-popup').dxButton({
+    text: 'Show Popup',
+    width: 300,
+    type: 'default',
+    onClick() {
+      popupWithScrollableContainer.show();
```

Comment:
Naming also affects the code, not only a markup.
```suggestion
      popup.show();
```

Reply:
Fixed